cmake/app/boilerplate.cmake

```
@@ -245,6 +245,15 @@ if(${BOARD}_DEPRECATED)
 endif()
 
 foreach(root ${BOARD_ROOT})
+  # Check that the board root looks reasonable.
+  if(NOT IS_DIRECTORY "${root}/boards")
+    message(WARNING "BOARD_ROOT element without a \"boards\" subdirectory:
```

small nit.
You can use `'` instead and then there will be no need for escaping, like:
```suggestion
    message(WARNING "BOARD_ROOT element without a 'boards' subdirectory:
```

especially I find that easier for my eyes, when starting to read paths like:
```
\"/foo/bar/boards/<ARCH>/my_board\"
```

compared to:
```
'/foo/bar/boards/<ARCH>/my_board'
```

I always multi read lines with both `/` and `\` due to a certain OS and it's mess with path separators.

---

>  You can use `'` instead and then there will be no need for escaping

Done for both BOARD_ROOT and SOC_ROOT.